WorkloadManagementSystem/Agent/TaskQueueDirector.py

```diff
@@ -126,13 +126,13 @@
 __RCSID__ = "$Id$"
 
 import random, time, threading
-from DIRAC                                                       import S_OK, S_ERROR, List, Time, abort
+from DIRAC                                                       import S_OK, S_ERROR, gConfig, abort
```

Comment:
One per line? I have no strong feeling about it, in fact I don't care, but if this is in our conventions, I can try and make the effort ;-)